Jenkinsfiles/signature-verification.sh

```diff
@@ -53,10 +61,19 @@ COPY="nix copy --from https://cache.vedenemo.dev"
 [[ ! -e "$path_to_check" ]] && $COPY "$path_to_check"
 [[ ! -f "$signature_file" ]] && $COPY "$signature_file"
 
-
 # get the sha256 of the file being verified
 PATH_HASH="$(Calc_sha256sum "$path_to_check")"
 
+ # Check for valid input as xxd -r will silently ignore errors
+if [[ ! $PATH_HASH =~ ^[0-9A-Fa-f]{64}$ ]]; then
+    echo "${PATH_HASH}: Hash calculation failed" >&2
+    exit 1
+fi
+# Make temp directory and copy <hydra>.pem file there
+MYTEMP=$(mktemp -d)
+cp "$pem_file" /"$MYTEMP"
```

Comment:
shellcheck complains that pem_file is referenced but not assigned.

If this environment variable needs to be set for this script, it should be documented in the generic comments for this script file in the beginning of this file.

Also the shellcheck complaint should be explicitly silenced with a comment here.